### PR TITLE
added documentation for .env.example file

### PR DESCRIPTION
**Pull request by bogoevskig**

### CODE REVIEW

**Comment by ghost** [comment]
[**DeepCode Report (#ce187e)**](https://www.deepcode.ai/app/gh/leogregianin/brcep/5bf9f3f7bd9f4fe8cae56c23c8863b1087b61b72/leogregianin/brcep/ce187e1bdb7543dc8445b90a4bf05881f182a3c5/_/%2F/code/)
---
DeepCode analyzed this pull request.
There are no new issues.

File: docs/env.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Environment variables
+
+### Change server address
+    BRCEP_ADDRESS=:8000
```
Comment (ghophp) [inline docs/env.md:4]: why tabs at the start of the line?

Comment (bogoevskig) [reply docs/env.md:4]: the tabs are added to make the line look as a code block in the markdown preview

Comment (leogregianin) [reply docs/env.md:4]: This is flat config file, not need markdown format.

**Review by leogregianin — CHANGES_REQUESTED** [review]
Need to include "BRCEP_CEPABERTO_TOKEN" is a CepAberto API token and remove spaces at the beginning of each setting.

**Comment by leogregianin** [comment]
Thanks for all. Closed in #27

**Pull request closed without merging**